app/lib/submission_builder/state1099_int.rb

```diff
@@ -1,4 +1,5 @@
 module SubmissionBuilder
+  # @deprecated
   class State1099Int < SubmissionBuilder::Document
```

Comment:
what is this for?

Reply:
oops I was testing something, thanks for catching. 

for context, this is getting removed for this year, but is likely to come back, so I was trying out ways to make it clear to eng that it was intentionally pulled but will come back